### DISCUSSION TITLE
"The Pit" Leaderboard/Records

### DISCUSSION THREAD

**Discussion by ImTander** [nzp-team/nzportable#690] [Records and Speedruns]
Thought this would be a fun map to start a post for! :) 

![TPr24](https://github.com/nzp-team/nzportable/assets/161973907/a716bc2f-4ad7-4f81-82f4-f71ba856a109)

**Comment by DGxShAdOwZz** [comment]
![20240428151506_1](https://github.com/nzp-team/nzportable/assets/128944577/ca15c9c6-daef-4405-91fd-2697764eac0f)
decent start, map is hard though

**Comment by printhelloworldasdas** [comment]
im go to try xd

**Reply by printhelloworldasdas** [reply]
i failed but all i need do is practice

**Comment by Kingjaxgaming420** [comment]
![image](https://github.com/user-attachments/assets/042017d2-5a76-4935-b362-755820a5abfc)
Tied wr (died during a trade)

**Reply by Kingjaxgaming420** [reply]
will be uploading a video of the entire game :)
